superset/assets/javascripts/explore/components/controls/FilterControl.jsx

```diff
@@ -28,6 +28,7 @@ export default class FilterControl extends React.Component {
     }));
     this.state = {
       filters: initialFilters,
+      xhr: null,
```

Comment:
I would rename this to `activeRequest` or something more descriptive

Reply:
done.